<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupDialog</class>
 <widget class="QDialog" name="SetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setup</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>600</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>10</y>
     <width>390</width>
     <height>361</height>
    </rect>
   </property>
   <property name="title">
    <string>Camera</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Device:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelRegister">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>250</y>
      <width>241</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>(Optional) Select file containing register settings</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browseRegister">
    <property name="geometry">
     <rect>
      <x>319</x>
      <y>268</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pathRegister">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="partNumber">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelPartnumber">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>153</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Part Number (for JSON output)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="serialNumber">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>320</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelSerialnumber">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>300</y>
      <width>153</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Serial Number (for JSON output)</string>
    </property>
   </widget>
   <widget class="QListWidget" name="deviceList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>161</width>
      <height>201</height>
     </rect>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>201</width>
      <height>241</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="defaultPage">
     <widget class="QComboBox" name="bayer">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>181</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="qcamera">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>181</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="height">
      <property name="geometry">
       <rect>
        <x>116</x>
        <y>50</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="bitdepth">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labeldeviceID">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>90</y>
        <width>90</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Epiphan Device ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelBayer">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Bayer Pattern</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelHeight">
      <property name="geometry">
       <rect>
        <x>116</x>
        <y>30</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Height (pixels)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="width">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QLabel" name="labelBitdepth">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>67</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Bits per pixel</string>
      </property>
     </widget>
     <widget class="QComboBox" name="deviceID">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>110</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelWidth">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Width (pixels)</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelQcamera">
      <property name="geometry">
       <rect>
        <x>23</x>
        <y>140</y>
        <width>110</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Built-in Camera Name</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="dynamicDevicePage">
     <widget class="QSplitter" name="splitter">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QLabel" name="videoFormatLabel">
       <property name="text">
        <string>Video format</string>
       </property>
      </widget>
      <widget class="QComboBox" name="videoFormatComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>381</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Operator Console</string>
   </property>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>30</y>
      <width>89</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Full program path</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pathProgram">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="browseProgram">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>307</x>
      <y>48</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>80</y>
      <width>89</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Full path to ini file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browseIni">
    <property name="geometry">
     <rect>
      <x>307</x>
      <y>98</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pathIni">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="browseChart">
    <property name="geometry">
     <rect>
      <x>307</x>
      <y>148</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>130</y>
      <width>200</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Full path to Arbitrary Chart definition file</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pathChartDef">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
